### PR TITLE
Fix Rendering Error 

### LINKED ISSUE #34964: Reflectometry GUI doesn't render properly when switching tabs
**Describe the bug**

The Reflectometry GUI doesn't seem to be refreshing correctly when switching between tabs, ultimately making the GUI unusable. We think this started after the Qt version update.

Found during manual testing  - https://github.com/mantidproject/mantid/issues/34952#issuecomment-1384147236

**To Reproduce**

1. Open the Reflectometry GUI
1. Switch between the Runs and Experiment Settings tabs a few times

**Expected behavior**

The GUI should refresh to display the correct widgets for each tab.

**Screenshots**

This is the view that eventually results and the GUI must be closed and re-opened to continue working:

GUIRefreshError

**Platform/Version (please complete the following information):**

 We've seen this on Ubuntu 18 and IDAaaS. We haven't seen it on Windows and haven't yet tested on Mac.

### PR DESCRIPTION
**Pull request by cailafinn**
**Description of work.**
Pinned qt on conda to a slightly earlier patch to avoid the rendering issues seen in the instrument viewer, alf, and refl guis. 

**To test:**
1. Update your conda environment from the supplied mantid dev `.yaml`
2. Launch workbench
3. Open the refl GUI & instrument viewer and flick through the tabs. 
4. Check that the rendering is working as it should. 

Fixes #34964 

*This does not require release notes* because **it concerns our conda management.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by martyngigg** [comment]
@ConorMFinn Thanks for this. Could you change the `base` branch to `release-next`?

File: conda/recipes/conda_build_config.yaml [hunk 1]
```diff
@@ -42,6 +42,9 @@ sphinx_bootstrap_theme:
 pyqt:
   - 5.15
 
+qt_main:
+  - 5.15.4
```
Comment (martyngigg) [inline conda/recipes/conda_build_config.yaml:46]: As we pinned to a patch level could we add a comment here too?

**Review by martyngigg — CHANGES_REQUESTED** [review]

**Review by martyngigg — APPROVED** [review]
This looks great to me. Installing the test package uploaded to Anaconda results in qt-main 5.15.4 and not .6 being installed.

I noticed an issue if you just try an in-place upgrade an existing package and it *won't* downgrade Qt but we primarily need to fix the case of installing from scratch and this fixes that.

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Please can we merge this for this evening. The test failure on Windows is not related to these changes.

**Comment by DanielMurphy22** [comment]
Just as a note, pinning back to this minor version also helps to fix 2 other bugs: https://github.com/mantidproject/mantid/issues/35046 https://github.com/mantidproject/mantid/issues/35029

**Pull request merged**